### PR TITLE
feat: pull VERSION from git-semver

### PR DESCRIPTION
**Pull request by ernestojeda**
Signed-off-by: Ernesto Ojeda <ernesto.ojeda@intel.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -81,5 +89,44 @@ pipeline {
...
+    }
+
+    env.setProperty('VERSION', currSemver)
+    sh "echo '${currSemver}' > VERSION"
+
+    // sh 'cat VERSION' // debug
+    println "--> Current version is: ${env.VERSION}" // debug
 
+    currSemver
 }
```
Comment (bill-mahoney) [inline Jenkinsfile:132]: Is this functionality that can live inside edgex-global-pipelines somewhere to be reused across other Jenkinsfiles?

File: Jenkinsfile [hunk 2]
```diff
@@ -81,5 +89,44 @@ pipeline {
...
+
+        def currSemverArray = []
+        for(int i = 0; i < semverSplit.size(); i++) {
+            def ver = (i != semverSplit.size() - 1) ? semverSplit[i] : currVer
+            currSemverArray << ver
+        }
+        currSemver = currSemverArray.join('.')
+    }
+
+    env.setProperty('VERSION', currSemver)
```
Comment (soda480) [inline Jenkinsfile:125]: for (soon-to-be) consistency sake, can we use:
```
env.VERSION = currSemver
```
instead of setProperty?

Comment (ernestojeda) [reply Jenkinsfile:125]: Sure, I can do that

File: Jenkinsfile (hunk 1 above)
Comment (ernestojeda) [reply Jenkinsfile:132]: Eventually...I think we can bake it into the edgeXSemver function but I did not want to increase scope at the moment.

**Pull request merged**